docs/status.md

```diff
@@ -131,7 +131,7 @@ one of the following tracking labels.
 | round_nearest_afz        | yes           | yes          | yes            | yes             | no          |
 | round_nearest_even       | yes           | yes          | yes            | yes             | no          |
 | rsqrt                    | yes           | yes          | yes            | yes             | no          |
-| scatter                  | yes           | revisit      | no             | no              | no          |
+| scatter                  | yes           | revisit      | yes*           | no              | no          |
```

Comment:
"Type Inference" should be set to "yes" because we now have a spec for `scatter`, and this implementation satisfies it.